### PR TITLE
chore: update repo

### PR DESCRIPTION
**Pull request by ancoiscailletfr**

### CODE REVIEW

**Review by gregberge — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,9 +19,12 @@ Step-by-step tutorial on how to import an article into Sirius app with the Graph
 
 ## Playground
 
-GraphQL queries and mutations will be execute in Sirius GraphQL <a href="https://lemonde.sirius.press/api/graphql">playground</a>.
+GraphQL queries and mutations will be execute in Sirius GraphQL playground:
```
Comment (bocanico) [inline README.md:22]: `executed`

File: README.md [hunk 2]
```diff
@@ -161,36 +164,20 @@ mutation CreateRubric {
...
-}
-mutation CreateRubric {
-  createRubric(input: { key: "discuss", name: "discuss", main: false }) {
-    id
-    name
-  }
-}
 ```
 
+for each tags, for this exemple: "react", "showdev", "discuss"
```
Comment (bocanico) [inline README.md:169]: For this `example` we use "react", "showdev" and "discuss" tags.

File: README.md [hunk 3]
```diff
@@ -161,36 +164,20 @@ mutation CreateRubric {
...
-- "discuss": _bG9jYWw6VGFnOjM5MTY0_
+- "javascript": `bG9jYWw6VGFnOjM5MTYy`
+- "react": `bG9jYWw6VGFnOjM5MTYz`
+- "showdev": `bG9jYWw6VGFnOjM5MTYx`
+- "discuss": `bG9jYWw6VGFnOjM5MTY0`
 
 ### Author creation
 
-Execute the [createAuthor](https://lemonde.sirius.press/developer/docs/graphql-api/schema/#query) query to create new author named "DHANUSH N".
+Execute the `createAuthor` mutation to create new author named "DHANUSH N".
```
Comment (bocanico) [inline README.md:180]: create `a new`

File: src/index.js [hunk 4]
```diff
@@ -37,11 +37,11 @@ async function main() {
     })
 
     const hast = htmlToHast(article.body_html)
-    const imageIds = await importContentImages(hast)
+    //const imageIds = await importContentImages(hast)
```
Comment (bocanico) [inline src/index.js:40]: comment

**Pull request merged**